src/base/display.jl

```diff
@@ -1,13 +1,20 @@
-function Base.show(io::IO, drv::Driver)
-    drv.ptr == C_NULL && (return print(io, "NULL Driver"))
+function Base.show(io::IO, drv::Driver)::IO
+    if drv.ptr == C_NULL
+        print(io, "NULL Driver")
+        return io
+    end
     print(io, "Driver: $(shortname(drv))/$(longname(drv))")
+    return io
 end
 
-function Base.show(io::IO, dataset::AbstractDataset)
-    dataset.ptr == C_NULL && (return print(io, "NULL Dataset"))
+function Base.show(io::IO, dataset::AbstractDataset)::IO
+    if dataset.ptr == C_NULL
+        print(io, "NULL Dataset")
+        return io
+    end
```

Comment:
Again, possible `else`, but with all the stuff underneath here, maybe let it be?

Reply:
Yeah the [pattern](https://danp.net/posts/reducing-go-nesting/) was more for "check NULL, early return", so I'll let it be, thanks though!